web/views.py

```
@@ -305,6 +310,160 @@ def index(request):
+                            if choice_text.strip():
+                                Choice.objects.create(question=question, text=choice_text.strip())
+
+        return redirect("surveys")
+
+
+class SurveyDetailView(LoginRequiredMixin, DetailView):
+    model = Survey
+    template_name = "surveys/detail.html"
+    login_url = "/accounts/login/"
+
+    def get_context_data(self, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_context_data`

(ANN201)